modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyTree.jsx

```diff
@@ -42,14 +43,24 @@ import { REST_URL, MakeRequest } from "./util.jsx";
 //  vocabulary: Vocabulary info
 //  browseRoots: Boolean representing whether or not the vocabulary tree shows roots
 //  onCloseInfoBox: Callback to close term info box
+//  allowTermSelection: Boolean enabler for term selection from vocabulary tree browser
+//  initialSelection: Existing answers
+//  questionDefinition: Object describing the Vocabulary Question for which this suggested input is displayed
 //
 function VocabularyTree(props) {
-  const { open, path, onTermClick, registerInfo, getInfo, onClose, onCloseInfoBox, onError, browserRef, classes, vocabulary, browseRoots, ...rest } = props;
+  const { open, path, onTermClick, registerInfo, getInfo, onClose, onCloseInfoBox, onError, browserRef, classes, vocabulary,
+    browseRoots, allowTermSelection, initialSelection, questionDefinition, ...rest } = props;
 
   const [ lastKnownTerm, setLastKnownTerm ] = useState("");
   const [ parentNode, setParentNode ] = useState();
   const [ currentNode, setCurrentNode ] = useState();
   const [ roots, setRoots ] = useState(vocabulary.roots);
+  const maxAnswers = questionDefinition?.maxAnswers;
+  const addCheckbox = allowTermSelection && Number.isInteger(maxAnswers) && maxAnswers != 1;
+  const selectorComponent = addCheckbox ? Checkbox : Radio;
```

Comment:
Now all instances of the VocabularyBrowser have radio buttons by default (including filters and the admin page browser).

How about this:
```suggestion
  const selectorComponent = allowTermSelection ? (questionDefinition?.maxAnswers == 1 ? Radio : Checkbox) : undefined;
```
and in `VocabularyBranch` check if `selectorComponent is defined before rendering.